.taskcluster.yml

```diff
@@ -10,7 +10,7 @@ tasks:
     - $let:
           taskgraph:
               branch: taskgraph
-              revision: f949fb56d96b9509b071ba55853f2d60e8dbbf57
+              revision: a8366f88fc8b35b284ad550358c89ec10133fb42
```

Comment:
Thanks for updating this, it's good to have this up-to-date to benefit for all the under-the-hood goodies in taskgraph. I'll note though that this changes infrequently so shouldn't be concerned about pinning this constantly.